packages/mobile/src/screens/feed-screen/FeedScreen.tsx

```diff
@@ -72,7 +75,7 @@ export const FeedScreen = () => {
         loadMore={loadMore}
         refresh={handleRefresh}
         refreshing={isRefreshing}
-        selfLoad
+        selfLoad={!!signedIn}
```

Comment:
so for this basically it will self load all times except when initially signing in? or when it becomes true does that cause a selfLoad fetch only when we are actually signed in?

Reply:
Ok so I also ended up changing from `useEffectOnce` to `useEffect` in Lineup with a dependency on `selfLoad`. 

In the case where you are already signed in:
The lineup will render while the splash screen is showing. Then, once signedIn is true, it will fetch the data.

In the case where you aren't signed in yet:
The lineup will also render initially while the splash screen is showing, but is replaced by the sign on screen (this is due to the wonky logic I was referencing). Then, once signedIn is true, it will fetch the data.